apache/couchdb-fauxton
# Update documents/index-editor to use Redux

## Overview

Update documents/index-editor to use redux.

## Testing recommendations

- Create a new view
- Update a view
   - without Reduce
   - with a built-in Reduce
   - with a CUSTOM Reduce
- Update a view, change its name and the Design Doc
   - Verify the old view is deleted and the new one is created in the target ddoc

## Checklist
